maintenance/image-options/imageset.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Deploy images by digest
+description: Specify the digests the operator will use to deploy images.
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Deploy images by container registry digest for operator installations.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+Once released, official {{site.prodname}} images and tags will not be modified. However using an immutable digest allows specific images to be reviewed
+and verified by security teams.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+* **ImageSet**
+
+### Concepts
+
+#### Container registry
+
+A container registry provides access to container images referenced by tags or digest.
+
+#### Image tag
+
+Versioned container images are typically referenced by a tag which is appended to an image reference. Example: `<repo>/<image>:<tag>`. Container image tags are typically not expected be changed or updated, but this is not required or enforced by most image registries, meaning it is possible to push new code to the same image tag.
+
+#### Image digest
+
+Container images, when added to a container registry, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
+
+### Before you begin
+
+**Required**
+- Docker client is configured to pull images from the container registries where images are stored. 
```

Comment:
We should probably say this requires installation using the operator

Reply:
That info is in the "Big picture" but are you suggesting to include it here also?

Reply:
I created https://github.com/projectcalico/calico/pull/4437 with this change. After it is merged I'll pick that change to this PR also.

Reply:
Yeah, I missed that in the big picture. I think it's worth calling out in the "requirements" section.

Reply:
Updated